code/src/components/AddTodoForm.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import DatePicker from 'react-date-picker';
```

Comment:
remove ; for consistency

Reply:
Yes. Tried to not have any semicolons this time, but I see it snuck in there anyway :)